docs/NetworkAPI.md

```diff
@@ -0,0 +1,7 @@
+# NetworkAPI Documentation
+
+- [NetworkAPI Built-in Regions](docs/NetworkAPI_Regions.md)
```

Comment:
There should be` [NetworkAPI Built-in Regions](NetworkAPI_Regions.md)` without the `docs/`
because it is opened from /docs subpath already